Makefile

```diff
@@ -48,6 +52,12 @@ uninstall:
 
 installables: clean bootstrap
 	$(BUILD_TOOL) $(XCODEFLAGS) install
+	rm -rf "$(SWIFTLINTFRAMEWORK_BUNDLE)/Versions/Current/Frameworks/SourceKittenFramework.framework/Versions/Current/Frameworks"
+	xcrun swift-stdlib-tool --copy \
+		--scan-executable "$(SWIFTLINT_EXECUTABLE)" \
+		--scan-folder "$(SWIFTLINTFRAMEWORK_BUNDLE)" \
+		--platform macosx --destination "$(SWIFTLINTFRAMEWORK_BUNDLE)/Versions/Current/Frameworks" \
+		--strip-bitcode
```

Comment:
is it necessary to strip bitcode since we're only building on OS X, where `swiftc` shouldn't generate it?